tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SystemDataSqlClientTests.cs

```diff
@@ -67,9 +74,28 @@ public void SubmitsTraces(string packageVersion, string metadataSchemaVersion)
             using var process = RunSampleAndWaitForExit(agent, packageVersion: packageVersion);
             var spans = agent.WaitForSpans(expectedSpanCount, operationName: expectedOperationName);
 
-            Assert.Equal(expectedSpanCount, spans.Count);
+            spans.Count.Should().Be(expectedSpanCount);
             ValidateIntegrationSpans(spans, metadataSchemaVersion, expectedServiceName: clientSpanServiceName, isExternalSpan);
             telemetry.AssertIntegrationEnabled(IntegrationId.SqlClient);
+
+            // Testing that spans yield the expected output when using DBM
+
+            var settings = VerifyHelper.GetSpanVerifierSettings();
+            settings.AddRegexScrubber(new Regex("[a-zA-Z0-9]{32}"), "GUID");
+            settings.AddSimpleScrubber("out.host: (localdb)\\MSSQLLocalDB", "out.host: sqlserver");
+            settings.AddSimpleScrubber("out.host: sqledge_arm64", "out.host: sqlserver");
+
+            var fileName = nameof(SystemDataSqlClientTests) + $".Schema{metadataSchemaVersion.ToUpper()}";
+
+            fileName = fileName + (dbmPropagation switch
+            {
+                "service" or "full" => ".enabled",
```

Comment:
Shouldn't `"service"` and `"full"` generate different snapshots?

Reply:
And shouldn't those snapshots have the `/* ... */` comments added to the SQL query?

Reply:
Not really since for this snapshots we only test that `_dd.dbm_trace_injected` was added as expected and that the Spans did not yield errors after future changes we may do to this feature/integration, when it comes to the comment we inject itself is never in the span but only visible in DB container, I'll work on the system-tests to later also check that the query was commented as expected.

Reply:
Oh, I see. We change the `Command.CommandText` but not the resource name or the span tag. Thanks!